server/swagger.go

```diff
@@ -0,0 +1,21 @@
+package server
+
+import (
+	"net/http"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/gorilla/mux"
+	"github.com/rakyll/statik/fs"
```

Comment:
We don't need the `	_ "github.com/cosmos/cosmos-sdk/client/docs/statik"` import?

Reply:
I think so, I removed it since you'd mentioned it here [https://github.com/cosmos/cosmos-sdk/pull/12352#issuecomment-1173159878](url). Do you think its needed?

Reply:
I meant moving it back to another package. Have you tested this locally?

Reply:
bump @deepto98 :)